sdcm/utils/common.py

```diff
@@ -436,7 +436,7 @@ def inner(*args, **kwargs):
 
     def clean_up(self, futures):
         # if there are futures that didn't run  we cancel them
-        for future in futures:
+        for future, _ in futures:
```

Comment:
How did it work before? Is this code used at all?

Reply:
It was used mostly on lists